Validation/Geometry/test/.gitignore

```diff
@@ -0,0 +1,6 @@
+*.root
```

Comment:
@ianna, I think that if you want to add reference root files into this package, the data directory should be their natural place, not the test one. I'd rather prefer to leave this in.